File: VB/HowToUseCriteriaPropertyEditors.Win/WinApplication.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports DevExpress.ExpressApp.Win
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.ExpressApp

Namespace HowToUseCriteriaPropertyEditors.Win
    Partial Public Class HowToUseCriteriaPropertyEditorsWindowsFormsApplication
        Inherits WinApplication
        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub HowToUseCriteriaPropertyEditorsWindowsFormsApplication_DatabaseVersionMismatch(ByVal sender As Object, ByVal e As DevExpress.ExpressApp.DatabaseVersionMismatchEventArgs) Handles MyBase.DatabaseVersionMismatch
            e.Updater.Update()
            e.Handled = True
        End Sub
    End Class
End Namespace

File: CS/HowToUseCriteriaPropertyEditors.Win/WinApplication.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using DevExpress.ExpressApp.Win;
using DevExpress.ExpressApp.Updating;
using DevExpress.ExpressApp;

namespace HowToUseCriteriaPropertyEditors.Win {
    public partial class HowToUseCriteriaPropertyEditorsWindowsFormsApplication : WinApplication {
        public HowToUseCriteriaPropertyEditorsWindowsFormsApplication() {
            InitializeComponent();
        }

        private void HowToUseCriteriaPropertyEditorsWindowsFormsApplication_DatabaseVersionMismatch(object sender, DevExpress.ExpressApp.DatabaseVersionMismatchEventArgs e) {
            e.Updater.Update();
            e.Handled = true;
        }
    }
}
